examples/rust/devenv.nix

```diff
@@ -14,7 +16,9 @@
   #  clippy.enable = true;
   #};
 
-  packages = lib.optionals pkgs.stdenv.isDarwin (with pkgs.darwin.apple_sdk; [
+  packages = [
+    pkgs.wasm-pack
```

Comment:
Should we include that only of `wasm32-` target is set?

Reply:
I'm actually moving the example to `rust-wasm-cross` currently